src/main/java/Duke.java

```diff
@@ -5,61 +5,188 @@
+            System.out.print(String.format("%d. ", i + 1));
+            System.out.println(taskList[i]);
         }
         displayLine();
     }
 
-    //method adds task to task_list.
-    public static void addTaskToList(String task){
-        Task newTask = new Task(task);                  //create a new Task object.
-        taskList[taskCount] = newTask;                //add the new object to task_list.
-        taskCount++;                                   //keep track of the number of tasks in task_list.
-        System.out.println("added: " + task);
+    public static boolean isDeadline(String userInput){
+        if(userInput.length()>=8){
+            //return true if first 8 letters of userInput spell "deadline", else false
+            return userInput.substring(0,8).equals("deadline");
+        }
+        return false;
+    }
+
+    public static boolean isTodo(String userInput){
+        if(userInput.length()>=4){
+            //return true if first 4 letters of userInput spell "todo", else false
+            return userInput.substring(0,4).equals("todo");
+        }
+        return false;
+    }
+
+    public static boolean isEvent(String userInput){
+        if(userInput.length()>=5){
+            //return true if first 5 letters of userInput spell "event", else false
+            return userInput.substring(0,5).equals("event");
+        }
+        return false;
+    }
+
+    public static String getTodoFromUserInput(String userInput){
+        return userInput.substring(5,userInput.length());
+    }
+
+    public static String getEventFromUserInput(String userInput, int eventTimeIdx){
+        return userInput.substring(6,eventTimeIdx);
+    }
+
+    public static String getDeadlineFromUserInput(String userInput, int dueDateIdx){
+        return userInput.substring(9,dueDateIdx);
+    }
+
+    public static String getEventTimeFromUserInput(String userInput, int eventTimeIdx){
+        // ignore /by in the userInput
+        return userInput.substring(eventTimeIdx+4,userInput.length());
+    }
+
+    public static String getDueDateFromUserInput(String userInput, int dueDateIdx){
+        // ignore /at in the userInput
+        return userInput.substring(dueDateIdx+4,userInput.length());
+    }
+
+    public static void addTodoToList(String todoDescription){
+        //create a new Todo object.
+        Todo newTodo = new Todo(todoDescription);
+        //add the new object to taskList.
+        taskList[taskCount] = newTodo;
+        //keep track of the number of tasks in taskList.
+        taskCount++;
+        System.out.println("Noted. I've added:");
+        System.out.println(taskList[taskCount-1]);
+        System.out.println("Now you have "+Integer.toString(taskCount)+" tasks in the list.");
         displayLine();
     }
 
+    public static void addEventToList(String eventDescription, String eventTime){
+        // create a new Event object.
+        Event newEvent = new Event(eventDescription, eventTime);
```

Comment:
I like how you name an object. It follows the coding standard and is easier for readers to understand.